### ISSUE TITLE
Paired cohen.d with missing data

### ISSUE THREAD

**Issue by mtorchiano** [mtorchiano/effsize#9]
Invoking 
```
cohen.d(gr.1.measure, gr.2.t.measure, paired = T, na.rm = T)
```
saying that a paired text is not doable with NA's. However, I am specifying na.rm = T. Excluding the pairs involving NAs manually and re-doing cohen.d works fine.

The error is `"Paired computation requires equal number of measures."`

Maybe the code does not eliminate the pairs when only one item of a pair is missing

**Comment by mtorchiano** [comment]
There was a bug in the implementation

**Comment by mtorchiano** [comment]
Fixed in commit 2a0db66c26959589494be5cbfc8b910bf99a5f10
